README.md

```diff
@@ -1,45 +1,49 @@
-[![MIT License](https://img.shields.io/badge/license-MIT_License-green.svg?style=flat-square)](https://github.com/devopness/devopness/blob/main/LICENSE)
+git checkout -b <onboarding_daniel>
+[![MIT License](https://img.shields.io/badge/license-MIT-green.svg?style=flat-square)](https://github.com/devopness/devopness/blob/main/LICENSE)
 [![SDK JavaScript - CI](https://github.com/devopness/devopness/actions/workflows/sdk-javascript-ci.yml/badge.svg)](https://github.com/devopness/devopness/actions/workflows/sdk-javascript-ci.yml)
 
 # Devopness - Official open-source repository
 
 ## About Devopness
-`Devopness` aims to drastically simplify the way we manage cloud applications and multi cloud infrastructure in a secure and productive way.
+`Devopness` aims to drastically simplify the way we manage cloud applications and multi-cloud infrastructure in a secure and productive way.
 
-By streamlining essential DevOps practices we're making first class software deployment and cloud infrastructure management tools accessible and affordable to everyone in the world who is involved in software projects.
+By streamlining essential DevOps practices, we're making first-class software deployment and cloud infrastructure management tools accessible and affordable to everyone in the world who is involved in software projects.
 
-## 📚 <a id="docs"></a>Documentation
```

Comment:
What is the explanation for removing the 'a' tag?

Reply:
I returned the 'a' back to it's original place.